soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -1783,6 +1787,13 @@ void InitSaveEditor() {
         SohImGui::LoadResource(entry.second.name, entry.second.texturePath);
         SohImGui::LoadResource(entry.second.nameFaded, entry.second.texturePath, ImVec4(1, 1, 1, 0.3f));
     }
+    for (const auto& entry : gregMapping) {
```

Comment:
Do you need a map/loop here? We’re just loading in the two textures for Greg right?

Reply:
specifically it makes it easier to use the same patterns as the other groups when adding to the main window, instead of needing custom code just to add greg in there